packages/botonic-plugin-google-analytics/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@botonic/plugin-google-analytics",
+  "version": "0.14.0",
+  "main": "src/index.js",
+  "scripts": {
+    "cloc": "../../scripts/qa/cloc-package.sh .",
+    "prepare": "node ../../preinstall.js",
+    "lint": "npm run lint_core -- --fix",
+    "lint_ci": "npm run lint_core",
```

Comment:
clone .github/workflows/botonic-plugin-dashbot-tests.yml to create a github actions CI check

Reply:
Add a new hook to .pre-commit-config.yaml so that lint is executed when we do git commit